project/ProjectSettings.scala

```diff
@@ -32,7 +32,14 @@ object ProjectSettings {
     Compile / packageDoc / publishArtifact := false,
     Compile / doc / sources := Seq.empty,
     Compile / doc := target.map(_ / "none").value,
-    scalaVersion := "2.13.13",
+    scalaVersion := "2.13.14",
+    initialize := {
+      val _ = initialize.value
+      assert(
+        sys.props("java.specification.version") == "11",
+        "Java 11 is required for this project.",
+      )
+    },
```

Comment:
The JDK11 check was just removed 
- #27158

seems you pulled that back in now :wink: